threestudio/models/guidance/zero123_guidance.py

```diff
@@ -350,17 +350,17 @@ def guidance_eval(self, cond, t_orig, latents_noisy, noise_pred):
         # use only 50 timesteps, and find nearest of those to t
         self.scheduler.set_timesteps(50)
         self.scheduler.timesteps_gpu = self.scheduler.timesteps.to(self.device)
-        bs = latents_noisy.shape[0]  # batch size
-        large_enough_idxs = self.scheduler.timesteps_gpu.expand(
-            [bs, -1]
-        ) > t_orig.unsqueeze(
+        bs = min(4, latents_noisy.shape[0])  # batch size
```

Comment:
You're hardcoding the limit to 4, but in my PR merged yesterday, I made it configurable... please keep the configurability - I like to use 1 max_items, that makes it easier to compare vertically the results of different runs.

Reply:
I've made it configurable now, please check!